frontend/src/App.tsx

```diff
@@ -88,12 +88,47 @@ export default class App extends React.Component<Props, AppState> {
 
     private onMouseMove(e: React.MouseEvent<HTMLDivElement, MouseEvent>) {
         if (this.room) {
-            this.sendMessage({
-                messageType: "pointer_move",
-                playerId: this.room.sessionId,
-                pointerX: e.clientX,
-                pointerY: e.clientY,
-            });
+            let target = (e.target as HTMLElement | null);
+            // necessary? how to perform proper checks for cast?
+            if(target === null){
```

Comment:
I believe this is a more correct way, since it also makes sure that it is not another type of object.

```suggestion
            if(!(target instanceof HTMLElement)) {
```